### PR TITLE
Updated to support ISPManager Lite and Business.

### PR DESCRIPTION
**Pull request by JReissmueller**
Beefed up logging.

### CODE REVIEW

**Comment by abdyfranco** [comment]
I tried with different scripts, but I got the same error "MySQL Connection could not be established.". I don't know if it's only happening to me.

File: lib/ispmanager_installer.php [hunk 1]
```diff
@@ -44,78 +44,110 @@ public function install(stdClass $service, stdClass $meta)
...
+                [
+                    'request' => [
+                        'raw' => false,
+                        'follow_location' => true,
+                        'referer' => $softaculous_url . '?api=serialize&act=software'
+                    ]
                 ]
-            ]
-        );
+            );
```
Comment (abdyfranco) [inline lib/ispmanager_installer.php]: Moving the setOptions() call out of the else block, seems to fix the MySQL error.

**Review by abdyfranco — CHANGES_REQUESTED** [review]

File: lib/ispmanager_installer.php (hunk 1 above)
Comment (JReissmueller) [reply lib/ispmanager_installer.php]: Updated

**Comment by abdyfranco** [comment]
Everything seems to be working fine, but I think the version needs to be updated.

File: config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "version": "1.5.0",
+    "version": "1.5.2",
```
Comment (abdyfranco) [inline config.json:2]: Is there any reason why it should be 1.5.2 instead of 1.5.1? The latest stable version released is 1.5.0.

Comment (JReissmueller) [reply config.json:2]: The version was wrong in this file.  The project has already been tagged for 1.5.1

**Review by abdyfranco — APPROVED** [review]

**Pull request merged**